web-client/src/styles/tabs.scss

```diff
@@ -244,3 +244,7 @@
     margin-bottom: 0 !important;
   }
 }
+
+.tab-content-with-top-margin {
+  margin-top: 2.5rem;
+}
```

Comment:
We don't want the margin in the existing tabs, but do want a margin in the case list tabs. I added this class to accomplish this goal. If there is an existing class I can use, please let me know!